frontend/app/routes/protected/person-case/current-name.tsx

```diff
@@ -205,54 +248,81 @@ export default function CurrentName({ loaderData, actionData, params }: Route.Co
       </div>
       <Progress className="mt-8" label="" value={30} />
       <PageTitle subTitle={t('protected:in-person.title')}>{t('protected:current-name.page-title')}</PageTitle>
-      <p className="mb-8">{t('protected:current-name.page-description')}</p>
+      <p className="mb-4">{t('protected:current-name.recorded-name.description')}</p>
+      <ul className="mb-8 list-disc pl-5 font-bold">
+        <li>
+          {`${t('protected:current-name.recorded-name.first-name')} `}
+          <span className="font-normal">{loaderData.defaultFormValues.primaryDocsInfo.firstName}</span>
+        </li>
+        <li>
+          {`${t('protected:current-name.recorded-name.middle-name')} `}
```

Comment:
This might be a nitpick, but I think it's more correct to remove the space character from this string, and instead add a margin to the following span. ex: `<span className="ml-[1ch]">`

If you disagree, just mark this comment as resolved (it's not a blocker).